stdlib/LinearAlgebra/test/qr.jl

```diff
@@ -13,6 +13,7 @@ n2 = 2*n1
 
 Random.seed!(1234325)
 
+a = rand(n,n)
```

Comment:
Why do you need this extra matrix, when you could simply use `areal`? I'm asking because introducing it may mean that the subsequently randomly drawn matrices may change, even though the random seed was fixed.